usmqe_tests/api/gluster/test_volume.py

```diff
@@ -65,8 +65,7 @@ def test_create_volume_invalid(
     # TODO check correctly server response or etcd job status
     api.wait_for_job_status(
             job_id,
-            status="failed",
-            issue="https://github.com/Tendrl/tendrl-api/issues/33")
+            status="failed")
```

Comment:
These changes in test_volume file doesn't feel like part of 'Create cluster test'.
Just a note not an issue.